app/assets/javascripts/mapping.js

```diff
@@ -69,16 +28,24 @@ var MRConfig = (function () {
 (function() {
     var map;
     $(document).ready(function() {
-        var road_layer = new L.TileLayer('http://otile{s}.mqcdn.com/tiles/1.0.0/map/{z}/{x}/{y}.png', {
+        var osm_layer = new L.TileLayer('http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+            maxZoom: 18,
+            attribution: "&copy; <a href=\'http://openstreetmap.org\'> OpenStreetMap</a> contributors"
+        }),
+        road_layer = new L.TileLayer('http://otile{s}.mqcdn.com/tiles/1.0.0/map/{z}/{x}/{y}.png', {
             maxZoom: 18,
             subdomains: ['1', '2', '3', '4'],
             attribution: 'Tiles Courtesy of <a href="http://www.mapquest.com/" target="_blank">MapQuest</a>. Map data (c) <a href="http://www.openstreetmap.org/" target="_blank">OpenStreetMap</a> contributors, CC-BY-SA.'
         }),
-        satellite_layer = new L.TileLayer('http://otile{s}.mqcdn.com/tiles/1.0.0/sat/{z}/{x}/{y}.png', {
+        mapquest_layer = new L.TileLayer('http://otile{s}.mqcdn.com/tiles/1.0.0/sat/{z}/{x}/{y}.png', {
```

Comment:
the Mapquest layers allow only for a limited number of transactions (currently 15000) under the free plan. 
We can get free Skobbler / Telenav tiles for sure.
ESRI tiles in MR1 are used under a 'gentleman's agreement'. I'll need to confirm with them if we can continue using those. I'll create a separate ticket to track this.